src/components/Background.js

```diff
@@ -1,11 +1,43 @@
 import React from "react";
 import Header from './header/Header';
+import styles from './Background.module.css';
+import InfoBlock from "./info_block/InfoBlock";
 
 export default function Background({ data }){
+  const { label, education, experience } = data.page_spec_data;
+  console.log(education);
+  let testing = [...education];
+
+  const education_list = [];
+  for(let i = 0; i < 3; i++){
+    education_list[i] = education[Object.keys(education)[(1-i)]];
+  }
+  const experience_list = [];
+  for(let i = 0; i < 3; i++){
+    experience_list[i] = experience[Object.keys(experience)[(1-i)]];
```

Comment:
Check if this works with three or more experience block that have text. Because now there is only two and works with that but may not multiple.

Reply:
Next implementation works for three also.